homeassistant/components/lametric.py

```diff
@@ -0,0 +1,89 @@
+"""
+Support for LaMetric time.
+
+This is the base platform to support LaMetric components:
+Notify, Light, Mediaplayer
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/lametric/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['lmnotify==0.0.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_CLIENT_ID = 'client_id'
+CONF_CLIENT_SECRET = 'client_secret'
+
+DOMAIN = 'lametric'
+LAMETRIC_DEVICES = 'LAMETRIC_DEVICES'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_CLIENT_ID): cv.string,
+        vol.Required(CONF_CLIENT_SECRET): cv.string,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+
+# pylint: disable=broad-except
+def setup(hass, config):
+    """Set up the LaMetricManager."""
+    _LOGGER.debug("Setting up LaMetric platform")
+    try:
+        conf = config[DOMAIN]
+        hlmn = HassLaMetricManager(client_id=conf[CONF_CLIENT_ID],
+                                   client_secret=conf[CONF_CLIENT_SECRET])
+        devices = hlmn.manager().get_devices()
+    except Exception as exception:
```

Comment:
You can start with oauth and add more if you see others or other user report one